src/reversion/management/commands/deleterevisions.py

```diff
@@ -203,14 +217,14 @@ def handle(self, *app_labels, **options):
         # Delete versions and revisions
         if verbosity > 0:
             print("Deleting revisions...")
-        
+
         try:
             revision_query.delete()
         except DatabaseError:
             # may fail on sqlite if the query is too long
             print("Delete failed. Trying again with slower method.")
             for item in revision_query:
                 item.delete()
-                
```

Comment:
I have my editor configured to strip trailing whitespace. I tried to `git add -p` just the non-whitespace changes but some got through.